test/e2e/framework/network/utils.go

```diff
@@ -327,7 +328,20 @@ func (config *NetworkingTestConfig) DialFromContainer(ctx context.Context, proto
 		}
 		if responses.Difference(expectedResponses).Len() > 0 {
 			returnMsg := fmt.Errorf("received unexpected responses... \nAttempt %d\nCommand %v\nretrieved %v\nexpected %v", i, cmd, responses, expectedResponses)
+			// TODO(aojea) Remove once issues.k8s.io/123760 is solved
```

Comment:
just wondering if we should also log the routes and link before deleting pod netserver-0?
comparison will be easier, maybe?

Reply:
let's log one thing at a time, as Dan say below if there are sockets open they will be still  in TIME-WAIT